data/mods/gen7/random-teams.ts

```diff
@@ -29,568 +29,678 @@ const ZeroAttackHPIVs: {[k: string]: SparseStatsTable} = {
 	rock: {def: 30, spd: 30, spe: 30},
 };
 
+// Moves that restore HP:
+const RecoveryMove = [
+	'healorder', 'milkdrink', 'moonlight', 'morningsun', 'recover', 'recycle', 'roost', 'shoreup', 'slackoff', 'softboiled', 'strengthsap', 'synthesis',
+];
+// Moves that drop stats:
+const ContraryMoves = [
+	'closecombat', 'leafstorm', 'overheat', 'superpower', 'vcreate',
+];
+// Moves that boost Attack:
+const PhysicalSetup = [
```

Comment:
```suggestion
const PHYSICAL_SETUP = [
```

Reply:
this'll all require some cleanup but it shouldn't be too hard to fix it